clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
I was about to write a comment saying that this looks like it should be private to something, but then I decided to Google `defn-`, and whaddayaknow!

I do wonder if there's a way to encapsulate all these private functions into something more high level. Class-ish, but without the state.